lib/internal/process/main_thread_only.js

```diff
@@ -154,6 +154,13 @@ function setupSignalHandlers(internalBinding) {
       delete signalWraps[type];
     }
   });
+
+  // re-arm pre-existing signal event registrations
+  // with this signal wrap capabilities.
+  for (var ev in process._events) {
```

Comment:
Using public API: `for (const ev of process.eventNames())`?

Reply:
fixed.